wpilibc/src/main/native/include/frc/apriltag/AprilTag.h

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <wpi/SymbolExports.h>
+
+#include "frc/geometry/Pose3d.h"
+
+namespace frc {
```

Comment:
This class needs a wpi::json forward declaration like the other serialization instances.